native/cocos/renderer/gfx-agent/BufferAgent.cpp

```diff
@@ -43,7 +43,7 @@ BufferAgent::~BufferAgent() {
         BufferDestruct,
         actor, _actor,
         {
-            CC_SAFE_DELETE(actor);
+            actor = nullptr;
```

Comment:
Indeed, can do nothing here. As `_actor` is an IntrusivePtr, it will be assigned to `actor` which will add reference count automatically.

Reply:
Shall we add .reset() to IntrusivePtr? It will be the same as all other smart pointers, such as shared_ptr, unique_ptr

Reply:
Yep, i think can add .reset(). I will create an issue for it. It has the same effect here.